src/base/settingsstorage.cpp

```diff
@@ -65,12 +65,12 @@ namespace
     QString mapKey(const QString &key)
     {
         static const MappingTable keyMapping = {
-
```

Comment:
I use this (and below) to improve readability. I separate the part that has a meaning, from the service part (variable declaration).